src/config/network/wpa_supplicant.md

```diff
@@ -3,7 +3,29 @@
 The `wpa_supplicant` package is installed by default on the base system. It
 includes utilities to configure wireless interfaces and handle wireless security
 protocols. To use wpa_supplicant, you will need to enable [the wpa_supplicant
-service](#the-wpa_supplicant-service).
+service](../services/index.md#enabling-services). 
+
+Then you need to run `rfkill` to make sure your radio device is not blocked.
+
+Run `rfkill unblock id|type` if it is shown blocked.
```

Comment:
Information about `rfkill` should be added to the general network page, because it applies to `wpa_supplicant` and IWD, as well as any other daemon we decide to add.

Reply:
It is already in, but we can perhaps expand it. https://docs.voidlinux.org/config/network/index.html